private/rules/maven_project_jar.bzl

```diff
@@ -17,12 +18,26 @@ def _maven_project_jar_impl(ctx):
     target = ctx.attr.target
     info = target[MavenInfo]
 
+    # Identify the subset of JavaInfo to include in the artifact
+    artifact_jars = calculate_artifact_jars(info)
+    artifact_srcs = calculate_artifact_source_jars(info)
+
     # Merge together all the binary jars
     bin_jar = ctx.actions.declare_file("%s.jar" % ctx.label.name)
-    _combine_jars(ctx, ctx.executable._singlejar, info.artifact_jars.to_list(), bin_jar)
+    _combine_jars(
+        ctx,
+        ctx.executable._singlejar,
+        artifact_jars,
+        depset(transitive = [ji.transitive_runtime_jars for ji in info.dep_infos.to_list()]).to_list(),
```

Comment:
This line looks like it's creating the transitive dep_info jars to be excluded from artifact_jars, where artifact_jars already excluded this same list of transitive_runtime_jars from `calculate_artifact_jars`. Am I misreading something or is this doing redundant work?